src/aviary/message.py

```diff
@@ -76,6 +83,17 @@ def serialize_content(cls, data):
 
         return data
 
+    @model_serializer(mode="wrap")
+    def maybe_serialize_info(self, handler, serialization_info: SerializationInfo):
```

Comment:
Can you add a comment to `Message.info`'s `exclude=True` kwarg just adding rationale why it's there